dotcom-rendering/src/web/components/FixedThrasher.tsx

```diff
@@ -0,0 +1,45 @@
+import { ArticleDesign } from '@guardian/libs';
+import { Card } from './Card/Card';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+export const FixedThrasher = ({ trails, containerPalette, showAge }: Props) => {
+	const thrasher = trails[0];
+
+	return (
+		<Card
```

Comment:
I find this relationship between snaps, cards and thrashers pretty hard to get my head around. I don't see that there's a lot of the `Card` code being used here... so... I wonder if there's a way to break this apart?